spec/lib/statistics_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Statistics do
+
+  def mom_changed_cases_growth(start_cases:, end_cases:)

```

Comment:
Seems like these could be helpful in the `Statistics` model itself, not just in the specs

Reply:
What, to basically have instance methods that act as a shell for class methods.  I'm not so sure if that's a good idea - I like being explicit about what module the method is coming from.